Source: http://sources.gentoo.org/cgi-bin/viewvc.cgi/gentoo-x86/mail-filter/libmilter/files/libmilter-build-system.patch
Upstream: no
Reason: make it possible to link to libmilter using -Wl,-as-needed
diff -ruN sendmail-8.14.9.orig/devtools/M4/UNIX/defines.m4 sendmail-8.14.9/devtools/M4/UNIX/defines.m4
--- sendmail-8.14.9.orig/devtools/M4/UNIX/defines.m4	2014-03-06 18:31:29.000000000 +0100
+++ sendmail-8.14.9/devtools/M4/UNIX/defines.m4	2014-10-04 21:22:26.938780910 +0200
@@ -26,7 +26,7 @@
 CCLINK = ifdef(`confCCLINK', `confCCLINK', `confCC')
 # Linker for libraries
 LD=	ifdef(`confLD', `confLD', `confCC')
-LDOPTS=	ifdef(`confLDOPTS', `confLDOPTS') ifdef(`confMT', ifdef(`confMTLDOPTS', `confMTLDOPTS', `'), `')
+LDOPTS=	ifdef(`confLDOPTS', `confLDOPTS')
 LDOPTS_SO= ${LDOPTS} ifdef(`confLDOPTS_SO', `confLDOPTS_SO', `-shared')
 
 # Shell
@@ -71,7 +71,7 @@
 LIBADD= ifdef(`conf_'bldCURRENT_PRD`_LIBS', `conf_'bldCURRENT_PRD`_LIBS')
 
 # libraries required on your system
-LIBS= ${LIBADD} ifdef(`confLIBS', `confLIBS') ifdef(`conf_'bldCURRENT_PRD`_LIB_POST', `conf_'bldCURRENT_PRD`_LIB_POST')
+LIBS= ${LIBADD} ifdef(`confLIBS', `confLIBS') ifdef(`conf_'bldCURRENT_PRD`_LIB_POST', `conf_'bldCURRENT_PRD`_LIB_POST') ifdef(`confMT', ifdef(`confMTLDOPTS', `confMTLDOPTS', `'), `')
 
 # location of sendmail binary (usually /usr/sbin or /usr/lib)
 BINDIR=	ifdef(`confMBINDIR', `confMBINDIR', `/usr/sbin')
